images/README.md

```diff
@@ -5,17 +5,10 @@ that we use to run a bunch of our test and release scripts.
 
 ## To update the test worker images used in the Tekton tasks
 
-1. Build a new image.
```

Comment:
How is this done now?

Reply:
Not sure I follow; its done using the `make command` listed here which performs the steps I mentioned above and which were listed here in the README.